### ISSUE TITLE
INT_MAX domain issue

### ISSUE THREAD

**Issue by aowen87** [visit-dav/live-customer-response#15]
Dear friends,

   I am trying to run VisIt 3.1.1 (from home, due to our situation of
confinement), and I am getting the following problem:

  The compute engine running on host oliba.uv.es issued the following
warning: Mesh and/or variable size exceeds INT_MAX for domain 0. This
domain is being set to empty. Your only recourse to correct this issue
is to decompose your input database into smaller domains. If you
continue to process this data, this and likely other warnings will
periodically re-appear...

   The grid is 1024x20148x1024. I have been able to map a
1024x1024x1024 grid, but I get that problem for the bigger one. What
could I do to solve it? Thanks a lot in advance. Take care.

      Manel Perucho.

--
Manel Perucho Pla
Departament d'Astronomia i Astrofísica / Observatori Astronòmic
Universitat de València
e-mail: manel.perucho <at> valencia.edu
web:http://www.uv.es/perucho
tel.: +34 96 354 33 63

CLÀUSULA DE CONFIDENCIALITAT/CONFIDENTIALITY CLAUSE
(ACGUV 227/2014 Art. 7.1)
(http://links.uv.es/E0GKasq,http://links.uv.es/26uC3dX)/
(http://links.uv.es/BBz6HG5,http://links.uv.es/GAk63v0)

We're trying to [combine `visit-users` email list with GitHub issues](https://github.com/visit-dav/live-customer-response/wiki/How-the-new-GitHub-visit-users-Email-Integration-Works).
When replying on `visit-users`, please reply **only** to emails with a *Subject*
line that includes `[visit-dav/live-customer-response]`.

---

**Comment by aowen87** [comment]
Hi Manel, 

What data type are you reading into VisIt? It sounds like the entire datasets is contained on a single domain; is that correct?

Alister

**Comment by brugger1** [comment]
Hi Eric, locally I did it parallel, but the engine does not work (certainly I must have done something wrong) remotely, and I run serial. The problem appeared in both cases (locally + parallel and remotely + serial). 
 
 Thanks!
  
  Manel

**Comment by brugger1** [comment]
Hi Manel,

One other quick question. What file format reader is reading the data? VisIt has a lot of file formats based on HDF5, so knowing which reader is being used is helpful. If you open the file and then bring up the "File information" window (available under File->File information ...), that information should be near the top.

Thanks,

Eric

**Comment by brugger1** [comment]
Hi Eric, 
 
  this is what the file information says (first lines). Hope this is what you need. Thanks!  
  
   File = localhost:/scr/perucho/OUTPUTJ311RAA-00077.h5
==============================================================================
Database: /scr/perucho/OUTPUTJ311RAA-00077.h5
Simulation: No
Database comment: 
File format: Pixie_1.0
Num Time States: 1
MetaData is NOT repopulated on state changes
useCatchAllMesh: 0
Format  can do its own domain decomposition
Temporal extents are from 0 to 0,  dt = 0.
All Times are Accurate
Times: Are identical to cycles
All Cycles are Accurate
Cycles:

**Comment by brugger1** [comment]
Hi Manel,

That is exactly the information I need. That should do it on questions.

Eric

**Comment by brugger1** [comment]
Hi Manel,

I looked at the Pixie reader and it will split the block into multiple blocks when run in parallel. For example, if you are running on 8 processors, it will split it into 8 blocks, one per processor. In your case you could run on 2 processors and it should drop it below the limit. I tried this out on a small mesh with 8 processors and it split it into 8 pieces.

Eric

**Comment by brugger1** [comment]
Thanks a lot, Eric. I'll try that asap. Take care.

**Comment by markcmiller86** [comment]
Hi Manel,

Did using the Pixie reader work fo you?

**Comment by brugger1** [comment]
I'm closing this now since I provided a solution.
